File: JavaScript Básico/Sesión 3/Ejercicio2.js
for(let numero = 0; numero <=10; numero++)
{  
let resultado = true
  for (var i = 2; i < numero; i++) {
    if (numero%i==0){
      resultado = false;
    }
  }
  if (numero == 0 || numero == 1 )
    resultado = false
  if(resultado){
    console.log("Es número primo")
  }
  else
  {
    console.log(numero)
  }
}

File: Java Básico/Ejercicio Sesión 2/ejercicio2.java
import java.util.Scanner;
//Para este ejercicio tendréis que crear una función que reciba un precio y devuelva el precio con el IVA incluido.
public class Ejercicio2 {
    public static void main(String[] args) {
        System.out.println("El resultado con IVA es: "+calculariva());
    }

    private static double calculariva() {
        Scanner scanner = new Scanner(System.in);

        System.out.println("Introduce un numero: ");

        int numero = scanner.nextInt();

        double iva = numero * 0.21;

        return numero + iva;
    }
}
